wallet/src/state.rs

```diff
@@ -585,10 +592,10 @@ mod serde_helpers {
         note_commitment_tree: Vec<u8>,
         nullifier_map: Vec<(String, String)>,
         unspent_set: Vec<(String, String)>,
-        #[serde(default)]
-        pending_set: Vec<(String, SystemTime, String)>,
-        #[serde(default)]
-        pending_change_set: Vec<(String, SystemTime, String)>,
+        #[serde(default, alias = "pending_set")]
```

Comment:
This alias allows us to continue to parse wallets from before this change, without needing to run `pcli wallet reset`.